pkgs/servers/monitoring/uptime-kuma/default.nix

```diff
@@ -0,0 +1,56 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, fetchzip, substituteAll, nixosTests, iputils }:
+let
+  deps = import ./composition.nix { inherit pkgs; };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "uptime-kuma";
+  version = "1.18.0";
+
+  src = fetchFromGitHub {
+    owner = "louislam";
+    repo = "uptime-kuma";
+    rev = finalAttrs.version;
+    sha256 = "sha256-XcHgwYswkaqKrRWxE2TsLEZHQN535boPMhaLWxV3pn8=";
+  };
+
+  patches = [
+    # Fixes the permissions of the database being not set correctly
+    # See https://github.com/louislam/uptime-kuma/pull/2119
+    ./fix-database-permissions.patch
+  ];
+
+  postPatch = ''
+    substituteInPlace server/ping-lite.js \
+      --replace "/bin/ping" "${iputils}/bin/ping" \
+    --replace "/sbin/ping" "${iputils}/bin/ping" \
+    --replace "/sbin/ping6" "${iputils}/bin/ping"
+  '';
+
+  installPhase = ''
+    mkdir -p $out/
+    cp -r . $out/
```

Comment:
copying everything over to the output is super bloated, no need for `src` taking up `1.4MB`, `package-lock.json` taking up `1.3MB` and some other smaller bits

Reply:
And while it's not **intended** to be installed globally in system packages a proper location like `$out/share/uptime-kuma` might be better than filling `$out/`

Reply:
Done :)